tests/readrows_test.go

```diff
@@ -447,3 +447,55 @@ func TestReadRows_NoRetry_MultipleRowRanges(t *testing.T) {
 	res := doReadRowsOp(t, server, &req, nil)
 	assert.Len(t, res.Row, 4)
 }
+
+// TestReadRows_NoRetry_ClosedStartUnspecifiedEnd tests that the client can request
+// a row range with a closed start key and no end key.
+func TestReadRows_NoRetry_ClosedStartUnspecifiedEnd(t *testing.T) {
+	type testRow struct {
+		key, cf string
+	}
+
+	trs := []testRow{
```

Comment:
I wonder if you can just make the keys and values inline (i.e., remove the struct definition and the struct array).

After all, there are just two rows. So the for loop on line 471 is less straightforward than what you did in the other test cases.

Reply:
Done.